homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -62,7 +63,7 @@
         vol.Optional(CONF_ABOVE): vol.Coerce(float),
         vol.Optional(CONF_BELOW): vol.Coerce(float),
         vol.Required(CONF_P_GIVEN_T): vol.Coerce(float),
-        vol.Required(CONF_P_GIVEN_F): vol.Coerce(float),
+        vol.Optional(CONF_P_GIVEN_F): vol.Coerce(float),
```

Comment:
This has to be made optional again because otherwise the whole integration will fail at start up, before it has a chance to create an issue registry item